keps/sig-api-machinery/1965-kube-apiserver-identity/README.md

```diff
@@ -90,14 +90,14 @@ advertise IP address.
 
 ## Proposal
 
-We will use “hostname+PID+random suffix (e.g. 6 base58 digits)” as the ID.
-
 Similar to the [node heartbeats](https://github.com/kubernetes/enhancements/blob/master/keps/sig-node/589-efficient-node-heartbeats),
 a kube-apiserver will store its ID in a Lease object. All kube-apiserver Leases
-will be stored in a special namespace `kube-apiserver-lease`. The Lease creation
-and heart beat will be managed by a controller that is started in kube-apiserver's
-post startup hook. A separate controller in kube-controller-manager will be responsible
-for garbaging collecting expired Leases.
+will be stored in the `kube-system` namespace.
+
+The lease creation and heart beat
+will be managed by the `start-kube-apiserver-identity-lease-controller` post start hook
+and expired leases will be garbage collected by the `start-kube-apiserver-identity-lease-garbage-collector`
```

Comment:
It would be good to mention that this also has metrics associated with it.

Reply:
Added this one to the PRR section for metrics